lib/rubber/recipes/rubber/setup.rb

```diff
@@ -63,13 +63,14 @@ def enable_root_ssh(ip, initial_ssh_user)
     orig_password = fetch(:password)
     initial_ssh_password = fetch(:initial_ssh_password, nil)
 
-    if initial_ssh_user == 'ubuntu' || ENV.has_key?('RUN_FROM_VAGRANT')
-      set(:password, nil)
-    elsif initial_ssh_password
-      set(:password, initial_ssh_password)
-    else
-      set(:password, Capistrano::CLI.password_prompt("Password for #{initial_ssh_user} @ #{ip}: "))
-    end
+    set(:password, initial_ssh_password)
+    # if initial_ssh_user == 'ubuntu' || ENV.has_key?('RUN_FROM_VAGRANT')
```

Comment:
undecided can't restore the final else at the very least since it incorrectly prompts for a password.